### PR TITLE
Term Entry Java Calendar Complete

### PR DESCRIPTION
**Pull request by codeolly**
### Description

A Markdown file (complete.md) created to briefly explain the complete() method of the Calendar class in Java.
The document has a short explanation, then the syntax and an example.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/complete/complete.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- 'learn-java'
+- 'paths/computer-science'
+---
+
+The **complete()** method of the `Calendar` class is used to fill in any non-set fields in the calendar fields.
+
+## Syntax
+
+```pseudo
+protected void complete()
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/complete/complete.md:19]:
```suggestion
someCalendar.complete()
```

File: content/java/concepts/calendar/terms/complete/complete.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+Title: 'complete()'
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/complete/complete.md:2]:
```suggestion
Title: '.complete()'
```

File: content/java/concepts/calendar/terms/complete/complete.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+The **complete()** method of the `Calendar` class is used to fill in any non-set fields in the calendar fields.
+
+## Syntax
+
+```pseudo
+protected void complete()
+```
+
+> **Note:** If the time value has not been calculated from the calender field values, the computeTime() method is called. Then to calculate all calendar field values, the computeFields() method is called.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/complete/complete.md:22]:
```suggestion
> **Note:** If the time value has not been calculated from the calender field values, the `.computeTime()` method is called. Then to calculate all calendar field values, the `.computeFields()` method is called.
```

File: content/java/concepts/calendar/terms/complete/complete.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+## Example
+
+The example below demonstrates the use of the complete() method.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/complete/complete.md:26]:
```suggestion
The example below demonstrates the use of the `.complete()` method.
```

File: content/java/concepts/calendar/terms/complete/complete.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+```java
+import java.util.GregorianCalendar;
+public class CalendarCompleteExample extends GregorianCalendar {
+
+   public static void main(String[] args) {
+
+      // Create a new calendar
+      Main cal = new Main();
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/complete/complete.md:35]:
```suggestion
      CalendarCompleteExample cal = new CalendarCompleteExample();
```

File: content/java/concepts/calendar/terms/complete/complete.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+## Example
+
+The example below demonstrates the use of the complete() method.
+
+```java
+import java.util.GregorianCalendar;
+public class CalendarCompleteExample extends GregorianCalendar {
+
+   public static void main(String[] args) {
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/complete/complete.md:32]: all examples should use spaces(2) instead of tabs

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@codeolly I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by codeolly** [comment]
@caupolicandiaz thank you for the suggestions, all revisions have been made and commited again.

**Review by caupolicandiaz — APPROVED** [review]
@codeolly Looks good, this is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@codeolly Looks good!

**Pull request merged**